test/tests/jobs/CreateJobTest.cpp

```diff
@@ -349,42 +348,4 @@ struct CreateJobTest : tpunit::TestFixture {
         command["parentJobID"] = jobID;
         tester->executeWaitVerifyContent(command, "402 Auto-retrying parents cannot have children");
     }
-
-    // Retryable job (after retry period has passed) comes before QUEUED job
-    void retryJobComesFirst() {
```

Comment:
I removed this because this is basically testing `GetJob` and I think `testPrioritiesWithRunQueued` adequately covers this test case